core/templates/dev/head/base.html

```diff
@@ -56,7 +56,7 @@
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
     <title itemprop="name">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
-
+    <base href="/gallery">
```

Comment:
Though I think we can probably declare this along with the rest of the Oppia constants?